pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -402,11 +416,11 @@ an @tech{object}, so evaluating @racket[(f 7)] starts with a
 ]
 
 Unlike in algebra, however, the @tech{value} associated with an
-argument can be changed in the body of a procedure by using
+procedure parameter can be changed in the body of a procedure by using
```

Comment:
I thought that "procedure parameter" is such a standard term that it would be okay to use, particularly with the word "procedure" in front. Would "argument parameter" be okay? Or "argument parameter variable"?

Reply:
I think the other meaning of "parameter" (and its non-use in the sense of "formal parameter") is pretty deeply ingrained in the current documentation. If that's true, then I worry that "parameter" is going to be confusing, and "procedure parameter" is very close to the currently defined term "parameter procedure". It's possible that the terminology is not as ingrained and consistent as I think, though.

Reply:
I [feel your pain](https://beautifulracket.com/explainer/parameters.html) at not being able to use “parameter” in its generally accepted non-Racket sense. But I agree that Racket’s idiosyncratic but nonetheless ingrained use has skunked other uses of the term. Moreover, the problem is acute on this page, because Racket `parameter`s are discussed nearby.